tiatoolbox/models/engine/patch_predictor.py

```diff
@@ -1,31 +1,23 @@
-"""Defines Abstract Base Class for TIAToolbox Model Engines."""
+"""Defines PatchPredictor Engine."""
 
 from __future__ import annotations
 
 from typing import TYPE_CHECKING
 
-from typing_extensions import Unpack
-
 from .engine_abc import EngineABC, EngineABCRunParams
 
 if TYPE_CHECKING:  # pragma: no cover
-    import os
     from pathlib import Path
 
-    import numpy as np
     from torch.utils.data import DataLoader
 
-    from tiatoolbox.annotation import AnnotationStore
     from tiatoolbox.models.models_abc import ModelABC
-    from tiatoolbox.wsicore.wsireader import WSIReader
-
-    from .io_config import ModelIOConfigABC
 
 
 class PatchPredictor(EngineABC):
     r"""Patch level predictor for digital histology images.
 
-    The models provided by tiatoolbox should give the following results:
+    The models provided by TIAToolbox should give the following results:
 
     .. list-table:: PatchPredictor performance on the Kather100K dataset [1]
```

Comment:
should we report results in this class as well as in PatchClassifier?

Reply:
We are reporting these results for PatchClassifier.